spec/core.md

```diff
@@ -216,18 +205,18 @@ references the same identifier, regardless of the rest of the string that is the
 full `did:webs`, web searching could yield either the current location of the
 DID document, or a copy of the DID that may be useful. For example, even the [Internet
 Archive: Wayback Machine](https://archive.org/web) could be used to find a copy
-of the DID document and [[ref: KERI event stream]] at some point in the past that may be sufficient for the
+of the DID document and the [[ref: KERI event stream]] at some point in the past that may be sufficient for the
 purposes of the entity trying to resolve the DID. This specification does not
-rely on the Wayback Machine, but it might be a useful DID resolver tool.
+rely on the Wayback Machine, but it might be a useful `did:webs` discovery tool.
 
 The DID document, [[ref: KERI event stream]] and other files related to a DID may be copied to other web
 locations. For example, someone might want to keep a cache of DIDs they use, or
 an entity might want to run a registry of "useful" DIDs for a cooperating group.
 While the combination of DID document and [[ref: KERI event stream]] make the DID and DID document verifiable, just
-as when published in their "intended" location, the absence of DIDs equivalent to those locations
-in the DID document (`id` or `AlsoKnownAs`) means that the controller of the DID is
+as when published in their "intended" location, the absence of the `did:webs` in the [[ref: designated aliases]] for those locations
+in the DID document `equivalentId` means that the controller of the DID is
 not self-asserting any sort of tie between the DID and the location to which the
-DID-related documents have been copied. In contrast, if the controller confirms the link between the source and the copy with an `AlsoKnownAs`, the related copies will have to be looked after.
+DID-related documents have been copied. In contrast, if the controller confirms the link between the source and the copy with an `equivalentId`, the related copies will have to be looked after.
```

Comment:
"to be looked after" -> "to be kept in sync"?

Reply:
updated thx!